### PR TITLE
minor bugfixes/additions

### PR DESCRIPTION
**Pull request by Zett98**
### Bug fixes and small changes: 
- proper loading of wallet info(balance, threshold)
- replaced wrong favicon
- s/safe/wallet
- display balances in tez instead of mutez
- fix text color in various places
- add version information
- fix text in tx proposal form
- add indication for pending txes
- add indication for successful proposal submissions
- add ability to remove added wallets

### CODE REVIEW

File: components/create/createLoader.tsx [hunk 1]
```diff
@@ -16,6 +18,11 @@ function Success() {
         (async () => {
             if (loading && address.status == 0) {
                 try {
+                    const metadataMap = new MichelsonMap();
+                    metadataMap.set(
```
Comment (rueshyna) [inline components/create/createLoader.tsx:22]: Can you put `metadata.json` instead?
metadata.json can be found here.
https://github.com/marigold-dev/multisig/releases/tag/0.0.6

FYI: https://github.com/ligolang/multisig-cameligo/blob/main/deploy/deploy.ts#L78-L80

File: components/import/basic.tsx [hunk 2]
```diff
@@ -64,7 +64,7 @@ function Basic() {
                             <Field
                                 name="walletName"
                                 className=" border-2 p-2"
-                                placeholder="example-safe"
+                                placeholder="example-wallet"
```
Comment (rueshyna) [inline components/import/basic.tsx:67]: line 16 is still  `safe`

File: pages/wallets.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+                                return (
+                                    <Link
+                                        key={address}
+                                        className=" flex md:flex-row flex-col justify-between bg-primary font-medium text-white my-2 p-2 hover:bg-red-500 focus:bg-red-500 hover:outline-none border-2 hover:border-gray-800  hover:border-offset-2  hover:border-offset-gray-800"
+                                        href={`/wallets/${address}`}
+                                    >
+                                        {state.aliases[address] ?
+                                            <div>
+                                                <p className="text-xs md:text-xl">
+                                                    {state.aliases[address]} {state.contracts[address].version || "Unknown version"}
```
Comment (rueshyna) [inline pages/wallets.tsx:36]: version is in bytes.

File: pages/create.tsx [hunk 4]
```diff
@@ -47,7 +47,7 @@ function Home() {
             <div className="bg-graybg shadow">
                 <div className="mx-auto  max-w-7xl py-6 px-4 sm:px-6 lg:px-8 flex justify-start">
                     <h1 className="text-white text-2xl font-extrabold">
-                        Create multisig wallet
+                        Create multisig wallet (version 0.0.6)
```
Comment (rueshyna) [inline pages/create.tsx:50]: can we put version in config or in context?

**Review by rueshyna — COMMENTED** [review]
thanks for the fix :screwdriver: . I leave some comments.

File: components/create/createLoader.tsx (hunk 1 above)
Comment (Zett98) [reply components/create/createLoader.tsx]: fixed

File: components/import/basic.tsx (hunk 2 above)
Comment (Zett98) [reply components/import/basic.tsx:67]: fixed

File: pages/wallets.tsx (hunk 3 above)
Comment (Zett98) [reply pages/wallets.tsx:36]: fixed

**Pull request merged**